docs/magento2/search-engine.mdx

```diff
@@ -228,6 +229,102 @@ query Search {
 If you are using the default theme or the Chocolatine theme, the search bar
 should now be visible.
 
+## Attraqt
+
+> _This feature has been added in Front-Commerce `2.19.0`_
+
+### Requirements
+
+Due to the way Attraqt works, you will have to ensure your Attraqt index is
+Front-Commerce-ready before hand. In order to check this, we have added a simple
+script that check the basics for you, and tells what's missing from your current
+index.
+
+Using your Attraqt's search api key, you can execute this command from your
+Front-Commerce repository:
+
+```shell
+FRONT_COMMERCE_ATTRAQT_SEARCH_API_KEY=your_api_key npm run attraqt:check
+```
+
+:::info
+
+You can find your search API key by login in into you XO console, and navigating
```

Comment:
https://english.stackexchange.com/questions/5302/log-in-to-or-log-into-or-login-to
```suggestion
You can find your search API key by log in to your XO console, and navigating
```

A link to the XO console's targeted page may be good too (is it a generic URL ?)

Reply:
Typo noted !

> A link to the XO console's targeted page may be good too (is it a generic URL ?)

Unfortunately it isn't.